src/Indice.Psd2.Cryptography/Tokens/HttpMessageSigning/HttpSignatureDelegatingHandler.cs

```diff
@@ -100,6 +104,13 @@ HttpMessageHandler innerHandler
             return;
         }
 
+        /// <summary>
+        /// Ignores the Response Validation 
+        /// </summary>
+        public void IgnoreResponseValidation() {
```

Comment:
@dkarkanas Since there is no fluent configuration happening here I would say that a property is the most natural way to go here. 
```c#
public bool IgnoreResponseValidation { get; set; }
```